DuckDuckGo/TabViewController.swift

```diff
@@ -628,23 +631,24 @@ extension TabViewController: WKScriptMessageHandler {
         if event == "Request Allowed" {
             if let elapsedTimeInMs = dict["time"] as? Double,
                 let url = dict["url"] as? String {
-                instrumentation.request(url: url, allowedIn: elapsedTimeInMs)
+//                instrumentation.request(url: url, allowedIn: elapsedTimeInMs)
```

Comment:
This (as few below) is commented out to reduce impact on the performance, as there can be quite many requests that are allowed while browsing.